habitat_baselines/rl/ppo/policy.py

```diff
@@ -88,8 +89,11 @@ def forward(self, x):
         return self.fc(x)
 
 
+@baseline_registry.register_policy
 class PointNavBaselinePolicy(Policy):
-    def __init__(self, observation_space, action_space, hidden_size=512):
+    def __init__(
+        self, observation_space, action_space, hidden_size=512, **kwargs
```

Comment:
Maybe, worth to add ` **kwargs` to [PointNavResNetPolicy](https://github.com/facebookresearch/habitat-api/blob/master/habitat_baselines/rl/ddppo/policy/resnet_policy.py#L34).

Reply:
Done!